From e8abb2cb73db3d853c01833680786a7ea61c4df2 Mon Sep 17 00:00:00 2001
Message-ID: <e8abb2cb73db3d853c01833680786a7ea61c4df2.1696804840.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1696804840.git.sharlatanus@gmail.com>
References: <cover.1696804840.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sat, 7 Oct 2023 18:12:50 +0100
Subject: [PATCH 16/20] gnu: python-stdatamodels: Update packages style.

* gnu/packages/astronomy.scm (python-stdatamodels): Update indentation
to improve readability of the package.
[description]: Add TexInfo styling bits.
---
 gnu/packages/astronomy.scm | 55 ++++++++++++++++++++------------------
 1 file changed, 29 insertions(+), 26 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 7d54b4cb58..2587d17e99 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -2585,40 +2585,43 @@ (define-public python-stdatamodels
   (package
     (name "python-stdatamodels")
     (version "1.8.3")
-    (source (origin
-              (method url-fetch)
-              (uri (pypi-uri "stdatamodels" version))
-              (sha256
-               (base32
-                "0265xa9nrp6qp1z7v9kmnd3c8jc0dad2vzin657clk3rrsj2qll5"))))
+    (source
+     (origin
+       (method url-fetch)
+       (uri (pypi-uri "stdatamodels" version))
+       (sha256
+        (base32 "0265xa9nrp6qp1z7v9kmnd3c8jc0dad2vzin657clk3rrsj2qll5"))))
     (build-system pyproject-build-system)
     (arguments
      (list
-      ;; Disable tests requiring access to CRDS servers to download ~500MiB
-      ;; of data.
+      ;; Disable tests requiring access to CRDS servers to download ~500MiB of
+      ;; data.
       #:test-flags #~(list "-k" "not test_crds_selectors_vs_datamodel")
-      #:phases #~(modify-phases %standard-phases
-                   (add-before 'check 'set-home
-                     (lambda _
-                       (setenv "HOME" "/tmp"))))))
-    (propagated-inputs (list python-asdf
-                             python-asdf-astropy
-                             python-astropy
-                             python-numpy
-                             python-psutil))
-    (native-inputs (list python-crds
-                         python-pytest
-                         python-pytest-doctestplus
-                         python-scipy
-                         python-semantic-version
-                         python-setuptools-scm))
+      #:phases
+      #~(modify-phases %standard-phases
+          (add-before 'check 'set-home
+            (lambda _
+              (setenv "HOME" "/tmp"))))))
+    (propagated-inputs
+     (list python-asdf
+           python-asdf-astropy
+           python-astropy
+           python-numpy
+           python-psutil))
+    (native-inputs
+     (list python-crds
+           python-pytest
+           python-pytest-doctestplus
+           python-scipy
+           python-semantic-version
+           python-setuptools-scm))
     (home-page "https://github.com/spacetelescope/stdatamodels")
     (synopsis
      "Core support for DataModel classes used in calibration pipelines")
     (description
-     "Provides DataModel, which is the base class for data models implemented in
-the @acronym{JWST, James Webb Space Telescope} and @acronym{Roman, Nancy Grace
-Roman Space Telescope} calibration software.")
+     "Provides @code{DataModel}, which is the base class for data models
+implemented in the @acronym{JWST, James Webb Space Telescope} and
+@acronym{Roman, Nancy Grace Roman Space Telescope} calibration software.")
     (license license:bsd-3)))
 
 (define-public python-stpipe
-- 
2.41.0

